### DISCUSSION TITLE
lists support

### DISCUSSION THREAD

**Discussion by NonyaBe** [elk-zone/elk#1483] [Q&A] [answered]
What a great start.

Am using as my primary tool and have found it great for everything I need except for accessing lists. Unless I've missed it.

Will keep using with interest

**Comment by patak-cat** [comment, marked as answer]
Follow user lists progress here https://github.com/elk-zone/elk/issues/372, and you can already test basic functionality at https://main.elk.zone (deploys on each commit to main)
